pages/common/ccache.md

```diff
@@ -0,0 +1,24 @@
+# ccache
+
+> C/C++ compiler cache.
+> More information: <https://ccache.dev/manual/4.9.html#_command_line_options>.
```

Comment:
This will point to outdated documentation once a new release becomes available.
```suggestion
> More information: <https://ccache.dev/manual/latest.html>.
```